packages/terra-tabs/src/Tabs.module.scss

```diff
@@ -104,3 +104,4 @@
     }
   }
 }
+/* stylelint-enable */
```

Comment:
Let's move this after line 98 and change it to enable `selector-max-compound-selectors`.